torch/nn/functional.py

```diff
@@ -1904,15 +1904,15 @@ def interpolate(input, size=None, scale_factor=None, mode='nearest', align_corne
     `mini-batch x channels x [optional depth] x [optional height] x width`.
 
     The modes available for resizing are: `nearest`, `linear` (3D-only),
-    `bilinear` (4D-only), `trilinear` (5D-only), `area`
+    `bilinear`, `bicubic` (4D-only), `trilinear` (5D-only), `area`
```

Comment:
Module doc needs to be updated too.

Reply:
@SsnL  IIRC we decided to deprecate nn.interpolate(Upsample) module and only keep the functional one from the chat.